### ISSUE TITLE
Main page "GO Enrichment Analysis" 

### ISSUE THREAD

**Issue by lpalbou** [geneontology/geneontology.github.io#159]
@thomaspd @cmungall @pgaudet do we want to call that "Enrichment Analysis" since this is launching an over representation test ?

**Comment by cmungall** [comment]
cc @huaiyu we have discussed this on other tickets, my position has always been to use "Enrichment Analysis" as the generic term. This is consistent with how everyone I know in the community uses it, certainly how everyone in the GOC uses it. It's also consistent with statistical terminology.

Top rated answer here says the two are the same https://www.biostars.org/p/88926/

People are really confused about this, e.g. https://www.reddit.com/r/bioinformatics/comments/66374x/gsea_vs_go_enrichment/

I think the introduction of the generic-sounding Gene Set Enrichment Analysis (GSEA) to mean a very specific tool and method introduced a lot of confusion.

Let's use EA for the general class, and then for specific subtypes we just describe in plain english, e.g. enriching a set of genes using Fisher exact, enriching a ranked list of genes using rank test, etc

**Comment by pgaudet** [comment]
In this case, does the documentation Panther need to be changed (and even the names of the tests)? It's inconsistent with what @cmungall writes here.

**Comment by suzialeksander** [comment]
@pgaudet, to clarify: Panther's docs or our doc on Panther (http://geneontology.org/docs/go-enrichment-analysis/)?

**Comment by pgaudet** [comment]
I meant on the Panther site - if I recall correctly, the enrichment is described as the test where you list has quantitative expression values.

**Comment by pgaudet** [comment]
Looks like this is a 'wont fix'

@thomaspd @huaiyumi tagging you FYI
